PeerCo/small_scale_self_consumed_solar.md

```diff
@@ -0,0 +1,289 @@
+
+
+### Requirements on the operating context
+
+The project country shall meet the following criteria:
+1. delivery of micro-solar is not a requirement for building development or to meet an enforced carbon quota,
+2. emission intensity data is available at the hourly (or sub-hourly) level for the national or local grid.
+
+In cases where this methodology is used to generate carbon credits, extra requirements shall be met:
+
+3. PV arrays are not installed on more than 10% of all buildings in the market of operation
+
+
+## Project boundary
+
+GHG sources included or excluded from the project boundary.
+
+| Source                     | Gas     | Included | Justification           |
+| -------------------------- |---------|----------|------------------------ |
+| Grid electricity displaced | $CO_2$  | Yes      | Main emission source    |
+| Grid electricity displaced | $NO_x$, $SO_x$  | No       | Emissions data is not sufficiently granular and available the half hour or hour |
+
+
+## Measurement
+
+This section introduces the underlying data requirements.
+
+### Asset data
+
+PV production data shall be acquired via automatic digital communication to PeerCo's cloud once the project is established. However, in the initial phase of setting up a project (qualifying the site, its data and quantifying the benefits of the scheme), offline extractions of the data in an open format (which may be CSV, XLSX, etc.) may be accepted.
+
+All self-consumption data (and related generation data) shall be quality controlled prior its usage to generate verified certificates. This verification shall include:
+1. the maximum PV generation is in line with the PV asset's nominal rating,
+2. self-consumption is always lower than the total consumption,
+3. no significant PV generation is happening when there is no solar irradiance (e.g. at night).
+
+Where data is missing, filling of the gaps sall be allowed under the following conditions:
+1. data is not missing more than 5% of the time within a calendar month,
+2. gaps are filled with data whose daily profile and magnitude is consistent with the rest of the asset generation,
+3. if accumulated values are available for the relevant energy measurements, the sum of the generation over the missing data periods shall equate the generation indicated in the accumulated measurements.
+
+When carrying out this asset data verification, a tolerance should be given for numerical purposes (e.g. to account for rounding of measurements by digital systems). This tolerance shall however not be of a magnitude that significantly impacts the volume of the certificates generated.
+
+### Emission intensity data
+
+The emission intensity data used in calculations shall meet the following requirements:
+
+1. the time resolution of the data is hourly or higher,
+2. the geographical resolution of the data is country-level or higher,
```

Comment:
Reinforcing my earlier point - this language would allow for gaming, ie. pick local or national emissions factors based on which would earn the most $$$

Reply:
When selling EACs, the value is first and foremost going to be about the kWh, rather than the carbon (which acts as contextual data for buyers, and might provide a premium value). 
Ideally, we should indeed have very clear guidelines on this. However, it is tricky to enforce from an international perspective, as granularity hugely depends on the area (you can for example check [here](https://app.electricitymaps.com/map) comparing European countries where some have national, while others have sub-regions (e.g. Norway, Italy and Sweden) and states like Florida in the US where the resolution is much higher). 

We could include that the geographical resolution "should be as high as allowed by the data source" but even this leaves an open door to gaming and update difficulties if the granularity increases in the future (expecting to see an increasing amount of detail enabled in the coming years, if the current trends continue). On the other hand, enforcement with a "must be" would only make these difficulties worse (by invalidating the certificates after time of non compliance).

At this stage, we propose to address this by greater transparency (i.e. disclosing source and resolution) rather than complex criteria whose practicality is far from clear. See the update [here](https://github.com/wattcarbon/WEATS/pull/2/commits/c2dd5c05a383f25edde0ae8a8d880f7228d51674) as well as the rest of the list of information contained in the EAC